.eslintrc

```diff
@@ -1,42 +1,8 @@
 {
-  "extends": ["airbnb", "prettier"],
-  "env": {
-    "es6": true,
-    "browser": true,
-    "node": true,
-    "mocha": true,
-    "jasmine": true,
-    "protractor": true
-  },
-  "parser": "babel-eslint",
-  "rules": {
-    "prettier/prettier": [
-      "error",
-      { "trailingComma": "all", "singleQuote": true }
-    ],
-    "jsx-a11y/anchor-is-valid": [ "error", { "components": [ "Link" ], "specialLink": [ "to" ] } ],
-    "import/prefer-default-export": "off",
-    "react/prefer-stateless-function": 0,
-    "import/no-unresolved": 1,
-    "import/extensions": 0,
-    "import/no-extraneous-dependencies": 0,
-    "react/no-unused-prop-types": 0,
-    "no-plusplus": 0,
-    "no-nested-ternary": 0,
-    "react-hooks/rules-of-hooks": "error",
-    "react-hooks/exhaustive-deps": "warn"
-  },
-  "settings": {
```

Comment:
All the settings are required. If not, the ~/config root imports complains and the aliases also do.